testing/auto_lint.sh

```diff
@@ -1,20 +1,25 @@
 #!/bin/bash
 
-echo "==> STARTING ESLINT"
+reset="\e[0m"
+bold="\e[1m"
+green="\e[92m"
+yellow="\e[93m"
+red="\e[91m"
+
+echo -e "$bold$yellow==> STARTING ESLINT$reset"
 npm run lint
```

Comment:
Insert a space after this line

Reply:
fixed in commit [15a155e](https://github.com/vattenbagarna/Frontend/pull/9/commits/15a155ef54e4e93e7d1a154de299a232ea8da33b)